CustomDictionary.xml

```diff
@@ -71,6 +71,7 @@
       <Word>SingletonAttribute</Word>
       <Word>Prefetch</Word>
       <Word>ScopeId</Word>
+      <Word>Poco</Word>
```

Comment:
The checkin included an entry to CustomDictionary.xml for that, so it should be taken care of. I'm building clean.